### DISCUSSION TITLE
Oder create sdk design

### DISCUSSION THREAD

**Discussion by taijusanagi** [kaguraswap/monorepo#177] [General]
Hi, I'm working on order-related SDK.

I'm wrapping seaport/zeroEx order-create function into one SDK.
I want to get feedback on "how to give feedback to frontend when managing approval in create order process".

Currently, I'm building SDK that "when user call creates order, it process approval and sign".
However, do I have to separate this process and give separate feedback to the front?

**Comment by taijusanagi** [comment]
This is development task
[#178](https://github.com/kaguraswap/monorepo/issues/178)

**Comment by ksuhara** [comment]
I think we should separate and give feedback each time. 
- approval: link to explorer
- sign: route.push() to order page created

**Comment by taijusanagi** [comment]
Let's do it!
